### ISSUE TITLE
Explain the code meaning please.

### ISSUE THREAD

**Issue by roachsinai** [w-hat/ctci-solutions#6]
https://github.com/w-hat/ctci-solutions/blob/5e78d820f519e2c5e22c99c995263e3d77c9be9c/ch-08-recursion-and-dynamic-programming/11-coins.py#L19

**Comment by ngchwanlii** [comment]
Yes, please. It will be great for us to learn how this formula is derived.
